src/backend/tests/teams.test.ts

```diff
@@ -27,7 +27,8 @@ describe('Teams', () => {
   });
 
   test('getSingleTeam works', async () => {
-    jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(prismaTeam1);
+    const prismaTeam = { ...prismaTeam1, projects: [] };
```

Comment:
instead of having to do this all the time, we should make the team test data be of type `Prisma.TeamGetPayload<typeof teamQueryArgs>` so that it includes projects